src/ToDoList.jsx

```diff
@@ -9,19 +11,28 @@ export default function ToDoList({ toDoList, onRemoveTodo }) {
         </div>
     ) : (
         <div>
-            {toDoList.map(
-                (toDo) =>
-                    toDo.show && (
-                        <ToDo
-                            toDo={toDo}
-                            onRemoveTodo={onRemoveTodo}
-                            key={toDo.id}
-                        />
-                    ),
-            )}
+            {toDoList.map((toDo) => (
+                <Stack
+                    spacing={1}
+                    divider={<Divider orientation="horizontal" flexItem />}
+                    key={toDo.id}
+                >
+                    {toDo.show && (
+                        <ToDo toDo={toDo} onRemoveTodo={onRemoveTodo} />
+                    )}
+                </Stack>
+            ))}
         </div>
     );
 }
+ToDoList.propTypes = {
+    toDoList: PropTypes.PropTypes.arrayOf(PropTypes.object),
```

Comment:
should be an array of `ToDo` type.
you can use `PropTypes.PropTypes.arrayOf(ToDo.propTypes),` or export the ToDo type somewhere and import it here